tests/sentry/sentry_apps/api/bases/test_sentryapps.py

```diff
@@ -16,64 +20,100 @@
 from sentry.testutils.cases import TestCase
 from sentry.testutils.helpers.options import override_options
 from sentry.testutils.silo import control_silo_test
+from sentry.users.models.user import User
+
+
+# Ideally choose the most relevant endpoint to the tested functionality
+def make_drf_request(
+    request: HttpRequest, endpoint: Endpoint | None = None, *args: Any, **kwargs: Any
+) -> Request:
+    if not endpoint:
+        endpoint = Endpoint()
+    drf_request: Request = endpoint.initialize_request(request, *args, **kwargs)
+
+    return drf_request
```

Comment:
this function lowkey doesn't really do anything (i.e it just calls `initialize_request`) but save some of that whitespace, so could be down to remove (?)

Reply:
If we always have an `endpoint` then inlining the `initialize_request()` call seems reasonable to me.